app/cafe.py

```diff
@@ -0,0 +1,23 @@
+import datetime
+from app.errors import (OutdatedVaccineError,

```

Comment:
After opening the parentheses, write each the arguments on a new line, including the first one, the closing bracket is also written on a new line